sycl/test-e2e/README.md

```diff
@@ -320,3 +320,23 @@ implementation header files is still in progress and the final set of these
 "fine-grained" includes that might be officially documented and suggested for
 customers to use isn't determined yet. **Until then, code outside of this project
 must keep using `<sycl/sycl.hpp>` provided by the SYCL2020 specification.**
+
+## Marking tests as expected to fail
+
+Every test should be written in a way that it is either passed, or it is skipped
+(in case it is not compatible with an environment it was launched in).
+
+If for any reason you find yourself in need to temporary mark test as expected
+to fail under certain conditions, you need to submit an issue to the repo to
+analyze that failure and make test passed or skipped.
+
+Once the issue is created, you can update the test by adding `XFAIL` and
+`XFAIL-TRACKER` directive:
+```
+// GPU driver update caused failure
+// XFAIL: level_zero
+// XFAIL-TRACKER: intel/llvm#DDDDD
```

Comment:
Just wondering - can't we proceed with the full link instead? Looks like I need to go to the "issues", open some and then paste the number to find the ticket (is there another way?)

Reply:
The format allows full links here and, it is just that example uses a shortcut. I agree that it will be easier to have a full link, so let me make it stricter

Reply:
c908d656b6bcfd021466beb4c5d5a3ff8f5a91c7